source/auditing-logging.txt

```diff
@@ -164,52 +160,101 @@ In addition to the following examples, see the blogpost
    .. tab:: CLI
       :tabid: cli
 
-      Update Audit Configuration
-      ~~~~~~~~~~~~~~~~~~~~~~~~~~
+      Create and Enable Filter
+      ~~~~~~~~~~~~~~~~~~~~~~~~
 
-      Run the following {+atlas-cli+} command to audit all authentication events
-      for known users in your project:
+      The following document defines an audit filter that restricts audits to only the
+      authentication operations that occur against the ``test`` database. 
+      To learn more, see :manual:`Configure Audit Filters </tutorial/configure-audit-filters/>`.
 
-      .. include:: /includes/examples/cli-example-audit-logs-known-users.rst
+      .. include:: /includes/examples/cli-example-audit-filter.rst
 
-      Run the following {+atlas-cli+} command to audit a known user
-      via a configuration file:
+      To enable an audit filter, run the :ref:`atlas auditing update <atlas-auditing-update>`
+      command with the ``--enabled`` flag and specify the audit filter document in single quotes
+      to pass the document as a string: 
+
+      .. include:: /includes/examples/cli-example-audit-filter-use.rst
+
+      The following {+atlas-cli+} command enables an audit filter that is defined in 
+      the specified JSON configuration file: 
 
       .. include:: /includes/examples/cli-example-audit-logs-config-file.rst
 
+      Update Audit Configuration
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      To update your project's audit configuration, use the :ref:`atlas auditing update <atlas-auditing-describe>`
+      command and specify the new audit filter. The following command replaces the existing 
+      audit filter configuration with a new filter that audits all authentication events for known users in the project: 
+
+      .. include:: /includes/examples/cli-example-audit-logs-known-users.rst
+
       Describe Audit Configuration
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
       Run the :ref:`atlas auditing describe <atlas-auditing-describe>`
-      {+atlas-cli+} command to return the auditing configuration for the specified project:
+      command to return the auditing configuration for the specified project:
 
       .. include:: /includes/examples/cli-example-audit-logs-describe.rst
 
-      Create and Use Audit Filter
-      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      Retrieve Logs
```

Comment:
Question: I see you removed the command to retrieve the access log. Is there not a scenario where customers would want to see that? Should we not include that as well and explain the difference between the two?